config.json

```diff
@@ -1,48 +1,3 @@
 {
-  "source": ["data/tokens.json"],
-
-  "platforms": {
-    "Owlery": {
-      "transformGroup": "scss",
-      "buildPath": "build/scss/",
-      "files": [
-        {
-          "destination": "colors.scss",
-          "format": "scss/variables",
-          "filter": {
-            "type": "color"
-          }
-        },
-
-        {
-          "destination": "typography.scss",
-          "format": "scss/variables",
-          "filter": {
-            "type": "typography"
-          }
-        }
-      ]
-    },
-
-    "Aviary": {
-      "transformGroup": "js",
-      "buildPath": "build/ts/",
-      "files": [
-        {
-          "format": "javascript/es6",
-          "destination": "colors.ts",
-          "filter": {
-            "type": "color"
-          }
-        },
-        {
-          "format": "javascript/es6",
-          "destination": "typography.ts",
-          "filter": {
-            "type": "typography"
-          }
-        }
-      ]
-    }
-  }
+  "source": ["data/transformed-tokens.json"]
```

Comment:
Moved the platforms to `build.js` as done in the [last typography reformat MR](https://github.com/Fullscript/aviary-tokens/pull/13/files#diff-7d111b05fe05c50beb729d1e99a36010fbda8c89b57a1a26ac3e7b0778a0ed53)